README.md

```diff
@@ -28,42 +28,41 @@ import 'package:tizen_interop/4.0/tizen.dart';
 // Getting a string value from the Native API.
 // Prefer using `arena` to allocate memory because it frees the memory
 // automatically when the `using` block ends.
-final appName = using((Arena arena) {
-  Pointer<Pointer<Int8>> ppStr = arena();
+final String appName = using((Arena arena) {
+  final Pointer<Pointer<Char>> ppStr = arena();
   if (tizen.app_get_name(ppStr) == 0) {
     // The memory allocated by the Native API must be freed by the caller.
     arena.using(ppStr.value, calloc.free);
     return ppStr.value.toDartString();
   }
+  return 'unknown';
+});
+
+// Passing a string value to the Native API.
+// The memory allocated by the `toNativeChar` method must be freed by
+// the caller. The `arena` allocator will free it automatically.
+using((Arena arena) {
+  tizen.preference_set_int(
+      'tizen_interop_test_key_for_int'.toNativeChar(allocator: arena), 100);
 });
 
 // Getting an integer value from the Native API.
-final memSize = using((Arena arena) {
-  Pointer<Int32> pMemSize = arena();
-  if (tizen.runtime_info_get_physical_memory_size(pMemSize) == 0) {
-    return pMemSize.value;
-  }
+final int preferenceValue = using((Arena arena) {
+  final Pointer<Int> pValue = arena();
+  int ret = tizen.preference_get_int(
```

Comment:
`ret` can be `final`. You may either 

- Use `final` for `ret`.
- Omit `final` from all other values.
- Use `if (..)`.